src/llmq/instantsend.cpp

```diff
@@ -1179,6 +1191,15 @@ void CInstantSendManager::AddNonLockedTx(const CTransactionRef& tx, const CBlock
             ++it;
         }
     }
+
+    {
+        LOCK(cs_timingsTxSeen);
+        // Only insert the time the first time we see the tx, as we sometimes try to resign
+        if (auto it = timingsTxSeen.find(tx->GetHash()); it == timingsTxSeen.end()) {
+            timingsTxSeen[tx->GetHash()] = GetTimeMillis();
+        }
```

Comment:
let's avoid double lookup on map here:

```suggestion
        timingsTxSeen.try_emplace(tx->GetHash(), GetTimeMillis());
```

emplace() and try_emplace() are similar, but difference in performance - one is expecting success by default, other is expecting failure by default

Reply:
proof that it works:
```
$ cat a.cpp 
#include <iostream>
#include <unordered_map>

using namespace std;

int main() {
    unordered_map<int, int> m;
    if (m.try_emplace(2, 3).second) cout << "OK" << endl; else cout << "FAIL" << endl;
    if (m.try_emplace(3, 4).second) cout << "OK" << endl; else cout << "FAIL" << endl;
    if (m.try_emplace(2, 5).second) cout << "OK" << endl; else cout << "FAIL" << endl;
    for (auto i : m) {
        cout << i.first << ' ' << i.second << endl;
    }
}
$ ./a 
OK
OK
FAIL
3 4
2 3
```

Reply:
abcba593f4f32de65741dcc083072333530d03b4